### PR TITLE
[DST-363]: Adding Guideline-Page of how to use `Multiselection`

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

**Comment by OsamaAbdellateef** [comment]
now test is fixed  @sebald

**Review by sebald — DISMISSED** [review]

File: packages/components/src/TagGroup/Tag.tsx [hunk 1]
```diff
@@ -5,6 +5,9 @@ import { cn, useClassNames } from '@marigold/system';
 
 import { TagGroup } from './TagGroup';
 
+{
```
Comment (sebald) [inline packages/components/src/TagGroup/Tag.tsx:8]: I think this does not belong here? :D

File: packages/components/src/index.ts [hunk 2]
```diff
@@ -33,6 +33,7 @@ export * from './Link';
 export * from './List';
 export * from './Menu';
 export * from './Message';
+export * from './Multiselect';
```
Comment (sebald) [inline packages/components/src/index.ts:36]: Only a recipe, do not export as component.

**Review by sebald — CHANGES_REQUESTED** [review]
@OsamaAbdellateef is this now the only PR to merge? 

E.g. this includes the recipe and guide?

**Comment by OsamaAbdellateef** [comment]
> @OsamaAbdellateef is this now the only PR to merge?
> 
> E.g. this includes the recipe and guide?

Yeah it includes both @sebald

File: packages/components/src/index.ts (hunk 2 above)
Comment (sebald) [reply packages/components/src/index.ts:36]: please remove the line :) we don't want to expose this yet!

Comment (sebald) [inline packages/components/src/index.ts:36]:
```suggestion
```

**Review by sebald — DISMISSED** [review]

**Review by sebald — APPROVED** [review]

**Pull request merged**